src/dictionaries.py

```diff
@@ -35,10 +35,18 @@
 
 # Add a new waypoint to the list
 # YOUR CODE HERE
+waypoints.append(dict([('lat', 99), ('lon', -99), ('name', 'my place')]))
+# print(waypoints)
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # YOUR CODE HERE
+waypoints[0]['lon'] = -130
+waypoints[0]['name'] = "not a real place"
 
 # Write a loop that prints out all the field values for all the waypoints
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+for i in range(0,len(waypoints)):
```

Comment:
Python for loop is pretty smart.  You don't have to set a range if you are traversing through an object of a defined size.  It will simply pull out one component at a time and assign it to i.